diff --git a/source/heightmap.d b/source/heightmap.d
index 0b68f37..96ff7e6 100644
--- a/source/heightmap.d
+++ b/source/heightmap.d
@@ -349,6 +349,7 @@ class Heightmap
 	private Triangulation generateTriangulation()
 	{
 		vec2d[] points;
+		rndGen.seed(12345);
 		auto resolution = conf.resolution;
 
 		if (conf.grid == "square")
@@ -378,6 +379,12 @@ class Heightmap
 		else
 			throw new Exception("Unknown grid type: " ~ conf.grid);
 
+		{
+			import std.file : write;
+			import std.process : environment;
+			auto f = File(environment.get("WGEN_POINTS", "points.txt"), "w");
+			foreach(pt; points) f.writefln("%a %a", pt.x, pt.y);
+		}
 		return new Triangulation(points);
 	}
 
